go/vt/tabletserver/tx_executor.go

```diff
@@ -236,27 +254,33 @@ func (txe *TxExecutor) SetRollback(dtid string, transactionID int64) error {
 // ConcludeTransaction deletes the 2pc transaction metadata
 // essentially resolving it.
 func (txe *TxExecutor) ConcludeTransaction(dtid string) error {
-	defer txe.qe.queryServiceStats.QueryStats.Record("RESOLVE", time.Now())
+	defer txe.te.queryServiceStats.QueryStats.Record("RESOLVE", time.Now())
```

Comment:
Maybe this need to be moved after the:
if !txe.te.twopcEnabled {
so we don't record stats if 2pc is not enabled?

(this applies to the previous few methods too)